### PR TITLE
admin team user index

### PR DESCRIPTION
**Pull request by minhld1709**

### CODE REVIEW

File: app/controllers/admin/teams_controller.rb [hunk 1]
```diff
@@ -17,37 +17,39 @@ def new
...
-      flash[:danger] = t("admin.create.danger")
+      flash[:danger] = t "admin.create.danger"
       render :new
     end
   end
 
-  def update
+  def edit
     @team = Team.find params[:id]
+    @no_team_users = User.all.normal.no_team
```
Comment (dieunb) [inline app/controllers/admin/teams_controller.rb]: no need `all`

File: config/routes.rb [hunk 2]
```diff
@@ -24,6 +24,6 @@
       resource :positions
     end
     resources :positions
-    resources :teams
+    resources :teams
```
Comment (dieunb) [inline config/routes.rb]: remove trailing spaces

File: app/controllers/admin/teams_controller.rb [hunk 3]
```diff
@@ -17,37 +17,39 @@ def new
...
   end
 
-  def show
-    @team = Team.find params[:id]
-  end
-
   def destroy
     Team.find(params[:id]).destroy
-    flash[:success] = t('admin.destroy.success')
+    flash[:success] = t 'admin.destroy.success'
```
Comment (nghiabk) [inline app/controllers/admin/teams_controller.rb]: ' ' -> " "

File: app/controllers/admin/teams_controller.rb [hunk 4]
```diff
@@ -6,10 +6,10 @@ def index
     @teams = Team.paginate page: params[:page]
   end
 
-  def edit
+  def show
     @team = Team.find params[:id]
   end
-
+
```
Comment (dieunb) [inline app/controllers/admin/teams_controller.rb]: remove trailing spaces

File: config/routes.rb [hunk 5]
```diff
@@ -4,8 +4,8 @@
   get "about"   => "static_pages#about"
   get "contact" => "static_pages#contact"
 
-  devise_for :users
-
+  devise_for :users, skip: [:registrations]
+  resources :teams , only: [:edit, :update, :show, :index]
```
Comment (nghiabk) [inline config/routes.rb]: sử dụng except

File: app/controllers/admin/teams_controller.rb [hunk 6]
```diff
@@ -17,37 +17,39 @@ def new
...
     Team.find(params[:id]).destroy
-    flash[:success] = t('admin.destroy.success')
+    flash[:success] = t 'admin.destroy.success'
     redirect_to admin_teams_path
   end
 
   private
   def team_params 
-    params.require(:team).permit :name, :description, :leader
+    params.require(:team).permit :name, :description, :leader , user_ids:[]
```
Comment (nghiabk) [inline app/controllers/admin/teams_controller.rb]: space

File: config/routes.rb [hunk 7]
```diff
@@ -4,8 +4,8 @@
   get "about"   => "static_pages#about"
   get "contact" => "static_pages#contact"
 
-  devise_for :users
-
+  devise_for :users, skip: :registrations
+  resources :teams , except: [:new, :create]
```
Comment (nghiabk) [inline config/routes.rb]: remove space after teams

File: app/models/user.rb [hunk 8]
```diff
@@ -21,4 +22,7 @@ def is_admin?
     role == Settings.user.role.admin
   end
 
+  def is_normal?
```
Comment (phonghenry14) [inline app/models/user.rb]: dùng define method

Comment (HoangVanTrinh92) [inline app/models/user.rb]: Anh sử dụng define_method nhé.

**Pull request merged**